src/client/public/scripts/dash/dash.ts

```diff
@@ -1,5 +1,12 @@
-import {html, render} from '../../../../node_modules/lit-html/lit-html.js';
-import * as api from '../../../types/api';
+import {html, render} from '../../../../../node_modules/lit-html/lit-html.js';
+import * as api from '../../../../types/api';
+import { PullRequest } from '../../../../types/api';
+
+// Poll every 30 Minutes
+const POLLING_INTERVAL = 30 * 60 * 1000;
```

Comment:
30 seems really long. The dashboard needs to be pretty timely, so I would prefer if this was significantly shorter (in the order of minutes).

Reply:
5 Minutes? I just don't want to basically kill a users quota.

Reply:
5 is good for now. If this becomes problematic, we can look into reducing our query costs.